src/api/firebase.js

```diff
@@ -79,3 +85,10 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export async function checkListToken(listId) {
+	const listTokenQuery = query(collection(db, listId));
+	const listTokenSnapshot = await getDocs(listTokenQuery);
+
```

Comment:
@tinazarb @louieknolle Did you determine what was needed here via the Firebase docs?

Reply:
@ehammes yes we are querying the database to see if it the collection has documents associated with it as that's required, and it returns false if there aren't/collection doesn't exist